_posts/2019/2019-01-13-use-multiple-migrations-when-adding-database-constraints.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Use Multiple Migrations When Adding Database Constraints
+description: You need to avoid a long table rebuild and a write-lock
+layout: article
+category: ruby
+image:
+  base: '2018/use-multiple-migrations-when-adding-database-constraints'
+  alt: 'Locks'
+  source: 'https://unsplash.com/photos/8_NI1WTqCGY'
+  credit: 'marcos mayer'
+---
+
+Adding constraints to your application at the database layer is a good idea as it gives an extra layer of quality control to the data powering your application.
+
+Adding or changing default values, or even adding a new column with a default value to an existing table, is easy to do with Rails’ migrations. But beware...
```

Comment:
This sentence is a bit confusing leading on from the last one. Maybe you want to explain briefly want type of constraints you're talking about and then go into "beware"...?